src/Transports/MassTransit.EventHubIntegration/EventHubIntegration/ProcessorLockContext.cs

```diff
@@ -5,34 +5,37 @@ namespace MassTransit.EventHubIntegration
     using System.Threading.Tasks;
     using Azure.Messaging.EventHubs.Processor;
     using Checkpoints;
-    using MassTransit.Configuration;
+    using Logging;
     using Util;
 
 
     public class ProcessorLockContext :
-        IProcessorLockContext
+        IProcessorLockContext,
+        ProcessorClientBuilderContext
     {
         readonly SingleThreadedDictionary<string, PartitionCheckpointData> _data = new SingleThreadedDictionary<string, PartitionCheckpointData>();
-        readonly IHostConfiguration _hostConfiguration;
+        readonly ILogContext? _logContext;
         readonly ReceiveSettings _receiveSettings;
+        readonly CancellationToken _cancellationToken;
 
-        public ProcessorLockContext(IHostConfiguration hostConfiguration, ReceiveSettings receiveSettings)
+        public ProcessorLockContext(ILogContext? logContext, ReceiveSettings receiveSettings, CancellationToken cancellationToken)
```

Comment:
It lost my comment, but you can't do this you need to use hostConfiguration.ReceiveLogContext.